graylog2-web-interface/src/views/logic/fieldactions/ChangeFieldType/ChangeFieldTypeModal.tsx

```diff
@@ -132,9 +132,14 @@ const ChangeFieldTypeModal = ({ show, onClose, field }: Props) => {
                         required />
         </Input>
         <Alert bsStyle="info">
-          By default the type will be changed in all index sets of the current message/search. By expanding the next section, you can select for which index sets you would like to make the change.
+          By default the <b>{newFieldType}</b> as a field type for <b>{field}</b> will be changed in all index sets of the current message/search. You can select for which index sets you would like to make the change.
         </Alert>
         <IndexSetsTable field={field} setIndexSetSelection={setIndexSetSelection} fieldTypes={fieldTypes} initialSelection={initialSelection} />
+        <ControlLabel>Select rotation strategy
+          <HoverForHelp displayLeftMargin>
+            To see and use the <b>{newFieldType}</b> as a field type for <b>{field}</b>, you have to rotate indices. You can automatically rotate affected indices after submitting this form or do that manually later.
+          </HoverForHelp>
+        </ControlLabel>
```

Comment:
I was wondering why you are using the `ControlLabel` component here and in other cases, instead of a headline? Probably to make the  "Select Field Type For field" a label, which is connected with the input?

Since the label / headline is not connected to an input in all other cases, I was wondering if headlines make more sense.

I was also wondering why you used an alert for the "Select Field Type For controller" description and a popover for the "Select rotation strategy" description, don't they serve the same purpose?

I tried out using actual headlines (`h5`, because the modal headline seems to be `h4`, with a bottom margin of 5px) and paragraphs instead of an alert / popover and imo it looks a bit cleaner.

Before:
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/f1225410-7b08-4e22-b567-47f4342fcea1)


After:
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/d6de3aa6-f9d0-4815-81cf-3762dfd6c09b)

Reply:
I just saw that the "select new field type" description should be the description for the "select targeted index sets", but I think the point I am trying to make is still valid.

Reply:
We could still add a description for the "Select New Field Type" section like "Select a new field type for the **controller** field.", instead of mentioning the field type in the section headline ("Select Field Type For controller")